scout/adapter/mongo/variant.py

```diff
@@ -689,8 +699,13 @@ def get_region_vcf(
             for header_line in vcf_obj.raw_header.split("\n"):
                 if len(header_line) > 3:
                     temp.write(header_line + "\n")
-            for variant in vcf_obj(region):
-                temp.write(str(variant))
+            try:
+                for variant in vcf_obj(region):
+                    temp.write(str(variant))
+            except Exception:
```

Comment:
Are you sure you even reach this code, an exception alt line 679 should be raised, no?

Reply:
Yes we can reach this line without an index. IF we reach this line and it breaks we know that it is because the index is missing

Reply:
Ok!